plugins/block-sharable-procedures/src/events_procedure_parameter_create.ts

```diff
@@ -28,7 +30,7 @@ export class ProcedureParameterCreate extends ProcedureParameterBase {
   constructor(
       workspace: Blockly.Workspace,
       procedure: Blockly.procedures.IProcedureModel,
-      parameter: Blockly.procedures.IParameterModel,
+      parameter: ObservableParameterModel,
```

Comment:
This makes me sad.  😢

As discussed: why not generate a `VarCreate` event before each `ProcedureParameterCreate` event that that has the side-effect of creating a new variable, so that you can just look up the variable by name, rather than having to store its ID in the PPC event?